station.localization.json

```diff
@@ -4083,7 +4083,7 @@
       "ident_ios": "MenuTableViewController.None",
       "ident_android": "none",
       "en": "none",
-      "fi": "ei valittu",
+      "fi": "ei kirjautunut",
```

Comment:
@andreevdenis @markoaamunkajo this will replace all `none` strings in android app with `ei kirjautunut`. 

Is it ok @markoaamunkajo ?

Reply:
![2021-06-04_18-51-20](https://user-images.githubusercontent.com/722716/120811874-e63c1c80-c565-11eb-8716-4de20f635349.png)

@andreevdenis is this change ok with you?

Reply:
@rinat-enikeev thanks! in this case it's ok, since this string was used only in context of network activity.

Reply:
related to https://github.com/ruuvi/com.ruuvi.station.ios/issues/367#issue-911449000 (bottom of comment)